core/templates/dev/head/pages/collection-player-page/collection-player-page.directive.ts

```diff
@@ -250,6 +250,24 @@ oppia.directive('collectionPlayerPage', ['UrlInterpolationService',
               ctrl.collectionId);
           };
 
+          ctrl.getExplorationTitleStyle = function(index) {
+            var explorationTitleStyle = {
+              width: '160px',
+              position: 'absolute',
+              color: '#006553',
+              'font-size': '20px',
```

Comment:
I would suggest making this smaller - 18px, since the titles look quite big on your screenshots.

Reply:
Done!
@kevinlee12